### ISSUE TITLE
File input plugin: tailStartPosition doesn't work

### ISSUE THREAD

**Issue by radu-gheorghe** [sematext/logagent-js#132]
I'm not 100% sure if it's a bug or I'm doing something wrong.

I tried:
```
$ logagent --config /tmp/logagent.conf --tailStartPosition 0
...
2018-07-18T15:45:27.662Z pid[4994] Watching file:/tmp/a from position: 22
```

The same happened with a config like:
```
options:
  tailStartPosition: 0
```

I've also tried with -1 and it still reads from the end of the file.

**Comment by megastef** [comment]
Which operating system do you use?

**Comment by megastef** [comment]
It works for me on Mac OS X, Node 8.11.3, Logagent 2.0.112. Please install the latest Logagent version `npm i -g @sematext/logagent@latest`.

Please let me know version numbers. 
```
uname -a 
node  --version
logagent --version
```

**Comment by megastef** [comment]
Ok, it turned out the option `tailStartPosition` did not work, when used in the configuration file. 
Actually it was a problem in the config file loader. 
Fixed in commit: https://github.com/sematext/logagent-js/commit/a5a08237de584bbcc2586f592bc9b928621ca883

Released: + @sematext/logagent@2.0.113
